spec.md

```diff
@@ -478,18 +478,18 @@ message CreateVolumeRequest {
   // validating these parameters. COs will treat these as opaque.
   map<string, string> parameters = 5;
 
-  // End user credentials used to authenticate/authorize volume creation
-  // request.
+  // Credentials used by Controller plugin to authenticate/authorize
+  // volume creation request.
   // This field contains credential data, for example username and
   // password. Each key must consist of alphanumeric characters, '-',
   // '_' or '.'. Each value MUST contain a valid string. An SP MAY
   // choose to accept binary (non-string) data by using a binary-to-text
   // encoding scheme, like base64. An SP SHALL advertise the
   // requirements for credentials in documentation. COs SHALL permit
-  // users to pass through the required credentials. This information is
+  // passing through the required credentials. This information is
   // sensitive and MUST be treated as such (not logged, etc.) by the CO.
   // This field is OPTIONAL.
-  map<string, string> user_credentials = 6;
+  map<string, string> controller_credentials = 6;
```

Comment:
It's possible that the credentials used for `CreateVolume`/`DestroyVolume` are different than those used for `ControllerPublishVolume`/`ControllerUnpublishVolume`. Calling them `controller_credentials` makes it sounds like they should be the same. I think we either follow the suggestion in #140, or just name it as "credentials".

Reply:
I agree, let's go with just `credentials` or `create_credentials`, `delete_credentials`, `controller_publish_credentials`, etc.